packages/client/package.json

```diff
@@ -3,7 +3,7 @@
   "description": "An open-source app for building smarter, more agile teams.",
   "author": "Parabol Inc. <love@parabol.com> (http://github.com/ParabolInc)",
   "license": "AGPL-3.0",
-  "version": "4.4.0",
+  "version": "4.4.1",
```

Comment:
noticed that this is one minor version behind the others, this should be `4.4.2`